File: CS/Q139427/grid_control.cs
using DevExpress.XtraGrid;

namespace DXSample {
    public class MyGridControl : GridControl {
        public MyGridControl() : base() { }

        protected override void RegisterAvailableViewsCore(DevExpress.XtraGrid.Registrator.InfoCollection collection) {
            base.RegisterAvailableViewsCore(collection);
            collection.Add(new MyLayoutInfoRegistrator());
        }
    }
}

File: VB/Q139427/grid_control.vb
Imports DevExpress.XtraGrid

Namespace DXSample
    Public Class MyGridControl
        Inherits GridControl

        Public Sub New()
            MyBase.New()
        End Sub

        Protected Overrides Sub RegisterAvailableViewsCore(ByVal collection As DevExpress.XtraGrid.Registrator.InfoCollection)
            MyBase.RegisterAvailableViewsCore(collection)
            collection.Add(New MyLayoutInfoRegistrator())
        End Sub
    End Class
End Namespace